### PR TITLE
Use X-Shopify-Storefront-Access-Token header

### PR DESCRIPTION
**Pull request by swalkinshaw**
We want to standardize on our custom header instead of basic auth.

### CODE REVIEW

**Comment by swalkinshaw** [comment]
Note: do not merge yet

File: src/base64encode.js [hunk 1]
```diff
@@ -1,9 +0,0 @@
-export default function base64Encode(string) {
-  if (typeof btoa === 'function') {
-    return btoa(string);
-  } else if (typeof Buffer === 'function') {
-    return Buffer.from(string).toString('base64');
-  } else {
-    throw new Error('No native way to base64 encode');
-  }
-}
```
Comment (minasmart) [inline src/base64encode.js]: removing the need for this removes the need for a polyfill. Could you make sure `Base64` isn't in the package.json?

Comment (swalkinshaw) [inline src/base64encode.js]: Good catch, removed

**Comment by swalkinshaw** [comment]
Support for this header has shipped 🚀

**Comment by minasmart** [comment]
🚢  and tag as `alpha.3`

**Pull request merged**